File: Praktikum 6. Array/Soal 3/PRAK603-2310817310001-AvantioFierzaPatria.py
n1, n2 = map(int, input().split())

matrix1 = list(map(int, input().split()))

matrix2 = list(map(int, input().split()))

if n1 == n2:

    result = [matrix1[i] * matrix2[i] for i in range(n1)]
    print(*result)
else:

    print("jumlah tidak sama")

File: Praktikum 6. Array/Soal 3/PRAK603-2310817310001-AvantioFierzaPatria.c
#include <stdio.h>

int main()
{
    int n1, n2;

    scanf("%d %d", &n1, &n2);

    int matrix1[n1], matrix2[n2];

    for (int i = 0; i < n1; i++)
    {
        scanf("%d", &matrix1[i]);
    }

    for (int i = 0; i < n2; i++)
    {
        scanf("%d", &matrix2[i]);
    }

    if (n1 == n2)
    {

        for (int i = 0; i < n1; i++)
        {
            printf("%d ", matrix1[i] * matrix2[i]);
        }
        printf("\n");
    }
    else
    {

        printf("jumlah tidak sama\n");
    }

    return 0;
}
